src/Effects/CuNNy/CuNNy-2x4C-NVL.hlsl

```diff
@@ -0,0 +1,340 @@
+// CuNNy 2x4C BILINEAR RGB NVL - https://github.com/cunnyplapper/CuNNy
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+//!MAGPIE EFFECT
+//!VERSION 4
+//!SORT_NAME CuNNy-D04N02
+
+//!TEXTURE
+Texture2D INPUT;
+
+//!TEXTURE
+//!WIDTH INPUT_WIDTH * 2
+//!HEIGHT INPUT_HEIGHT * 2
+Texture2D OUTPUT;
+
+//!SAMPLER
+//!FILTER POINT
+SamplerState SP;
+
+//!SAMPLER
+//!FILTER LINEAR
+SamplerState SL;
+
+//!COMMON
+#define O(t, p) t.SampleLevel(SP, pos + p * pt, 0)
+#define V4 min16float4
+#define M4 min16float4x4
+
+//!TEXTURE
+//!WIDTH INPUT_WIDTH
+//!HEIGHT INPUT_HEIGHT
+//!FORMAT R8G8B8A8_SNORM
+Texture2D t0;
+
+//!TEXTURE
+//!WIDTH INPUT_WIDTH
+//!HEIGHT INPUT_HEIGHT
+//!FORMAT R8G8B8A8_SNORM
+Texture2D t1;
+
+//!PASS 1
+//!DESC CuNNy-2x4C-BILINEAR-RGB-NVL-in
```

Comment:
nit: The description is quite lengthy and may not fit well within the user interface. The prefix "CuNNy-2x4C-BILINEAR-RGB-NVL" seems superfluous in my view.

![image](https://github.com/Blinue/Magpie/assets/34770031/69b6ff7b-505f-42d9-a3db-22cfb2650c03)